packages/menu-bar/test/menu-bar.test.js

```diff
@@ -97,6 +97,27 @@ describe('root menu layout', () => {
     });
   });
 
+  it('should disable all buttons in the menu-bar', () => {
+    menu.disabled = true;
+    buttons.forEach((btn) => {
+      expect(btn.disabled).to.be.true;
+    });
+  });
+
+  it('should keep previously disabled buttons disabled when re-enabling the menu-bar', () => {
+    expect(buttons[2].disabled).to.be.true;
+    menu.disabled = true;
+    expect(buttons[2].disabled).to.be.true;
+    menu.disabled = false;
+    expect(buttons[2].disabled).to.be.true;
+  });
+
+  it('should also disable the overflow button', () => {
```

Comment:
This test seems to be redundant because it's covered by the test above (should disable all buttons).

Reply:
You're right, it is redundant. I thought it would be nice to explicitly test for this since it is part of the acceptance criteria. I will remove it.